Helper/Data.php

```diff
@@ -109,9 +123,14 @@ public function getTracingSampleRate(): float
 
     public function getIgnoreJsErrors(): array
     {
-        $list = $this->config['ignore_js_errors'] ?? [];
+        try {
+            $list = $this->serializer->unserialize($this->config['ignore_js_errors']);
```

Comment:
In case of the old deployment config this would throw errors as `$this->config['ignore_js_errors']` is already an array, breaking current installations.
It's the same case as getIgnoreExceptions https://github.com/justbetter/magento2-sentry/pull/121/files#diff-d52e7365b52ebb360796c9734ab3a62d2bd9f6097e38c253892c3d22a36668b5R395

I think it's best to check if the config value is not already an array (or specifically check if it is a string), and only if it is not. Unserialize the value

Reply:
I did something like this now, also to check for an null value, which is also an option.